bessctl/commands.py

```diff
@@ -502,6 +506,9 @@ def bind_var(cli, var_type, line):
         if re.match(r'^(\*|[_a-zA-Z][\w]*)$', val) is None:
             raise cli.BindError('"name" must be "*" or [_a-zA-Z][_a-zA-Z0-9]*')
 
+    elif var_type == 'fifo':
+        pass
+
```

Comment:
If 'fifo' is for already existing one, then how about setting var_candidates as existing filenames?